packages/ns-api/files/post-commit/reload-ipsets.py

```diff
@@ -17,5 +17,11 @@
             force_reload = True
             break
 
+if 'objects' in changes:
+    for change in changes['objects']:
+        if 'ipaddr' in change:
+            force_reload = True
+            break
```

Comment:
Let's force reload at any object change and call the script that reload all ipsets after fw4 restart:
```suggestion
if 'objects' in changes:
    force_reload = True
```